types/cordova-plugin-save-dialog/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "rules": {
+        "@definitelytyped/dt-header": "off",
+        "@typescript-eslint/prefer-namespace": "off"
```

Comment:
These should most definitely not be disabled.

Reply:
Ok, probably using `cordova-plugin-device-name` as a template has not been a good idea :laughing:

Reply:
Wow, I have absolutely no idea why so many packages have this rule disabled. Definitely not right.